website/client/components/inventory/stable/petItem.vue

```diff
@@ -78,6 +111,43 @@ div
       click () {
         this.$emit('click', {});
       },
+      getPetItemClass (pet) {
+        if (this.mountOwned && !this.isHatchable) {
+          return `GreyedOut Pet Pet-${pet.key} ${pet.eggKey}`;
+        }
+
+        if (this.isOwned) {
+          return `Pet Pet-${pet.key} ${pet.eggKey}`;
+        }
+
+        if (this.isHatchable) {
+          return 'PixelPaw';
+        }
+
+        if (this.mountOwned) {
+          return `GreyedOut Pet Pet-${pet.key} ${pet.eggKey}`;
+        }
+
+        // Can't hatch
+        return 'GreyedOut PixelPaw';
+      },
+    },
+    computed: {
+      potionClass () {
+        return `Pet_HatchingPotion_${this.item.potionKey}`;
+      },
+      eggClass () {
+        return `Pet_Egg_${this.item.eggKey}`;
+      },
+      isOwned () {
```

Comment:
computed properties like this seem used only in the getPetItemClass, why don't call the functions directly?

Reply:
only `mountOwned && isHatchable` in the vue, but I could change it to the method content too 

https://github.com/HabitRPG/habitica/pull/10768/files#diff-47365d041780039f2290fda1096a54feR8